Chindea Tudor Eugen/Lab3 CP/main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    // continue - este folosit in for  si cand se apeleaza se intoarce la inceput de for si creste contorul
+    // continue e poate folosi in orice functie repetitiva; te aduce la inceputul instructiunilor
+    int sequence[100],n,mini,pozmin[100],maxi,pozmax[100],ctmin=1,ctmax=1,i;
+    scanf("%d",&n);
+    for(i=0; i<n; i++)
+    {
+        scanf("%d",&sequence[i]);
+        if(i==0)
+        {
+            mini=sequence[i];
+            maxi=sequence[i];
+            pozmin[0]=i;
+            pozmax[0]=i;
+        }
+        else
+        {
+            if(sequence[i]>maxi)
+            {
+                maxi=sequence[i];
+                ctmax=0;
+                pozmax[ctmax]=i;
+            }
+            else if(sequence[i]==maxi)
```

Comment:
Well, if this is true, then the maximum number is still `maxi`, and I think you can simply leave it as it is, no need to do anything more about it